core/dbt/adapters/factory.py

```diff
@@ -114,8 +124,49 @@ def cleanup_connections(self):
             for adapter in self.adapters.values():
                 adapter.cleanup_connections()
 
-
-FACTORY: AdpaterContainer = AdpaterContainer()
+    def get_adapter_package_names(self, name: Optional[str]) -> List[str]:
+        if name is None:
+            # the important thing is that the global project is last.
+            return sorted(
+                self.packages,
+                key=lambda k: k == GLOBAL_PROJECT_NAME
+            )
+        package_names: List[str] = []
+        # package_names: Set[str] = {GLOBAL_PROJECT_NAME}
```

Comment:
Is this still here for posterity reasons?

Reply:
No, it's here because I forgot about it!